Community-Supported/hyper-jupyter-kernel/hyper_kernel/install.py

```diff
@@ -0,0 +1,55 @@
+import argparse
+import json
+import os
+import sys
+from shutil import copyfile
+
+from jupyter_client.kernelspec import KernelSpecManager
+from IPython.utils.tempdir import TemporaryDirectory
+from hyper_kernel.data import _data_root
+
+kernel_json = {
+    "argv": [sys.executable, "-m", "hyper_kernel", "-f", "{connection_file}"],
+    "display_name": "Hyper SQL",
+    "language": "sql",
+}
+
+
+def install_my_kernel_spec(user=True, prefix=None):
+    with TemporaryDirectory() as td:
+        os.chmod(td, 0o755)  # Starts off as 700, not user readable
+        with open(os.path.join(td, 'kernel.json'), 'w') as f:
+            json.dump(kernel_json, f, sort_keys=True)
+        copyfile(os.path.join(_data_root, 'logo-32x32.png'), os.path.join(td, 'logo-32x32.png'))
+        copyfile(os.path.join(_data_root, 'logo-64x64.png'), os.path.join(td, 'logo-64x64.png'))
+
+
```

Comment:
minor: why two blank lines?

Reply:
pep8...

The rest of the files didn't do it correctly. Adjusted them